docs/patch.md

```diff
@@ -290,11 +231,103 @@ Here's a minimal example showing how to decrement a value:
 patch.decrement('custom.score', 10);
 ```
 
+### remove
+
+This method removes values from a collection.
+
+The following table shows how the operation behaves in different scenarios:
+
+Current Value          | Given Value | Result
+-----------------------|------------ |-------------------
+`['a', 'b', 'c']`      | `'a'`       | `['b', 'c']`
+`['a', 'b', 'c', 'c']` | `'c'`       | `['a', 'b']`
+`['a', 'b', 'c']`      | `'d'`       | `['a', 'b', 'c']`
+`[]`                   | `'foo'`     | `[]`
```

Comment:
Is this example essential? It's the same as the above, isn't it?

Reply:
You're right, I did'nt even notice '^^